docker/runtime-base/build.gradle

```diff
@@ -0,0 +1,49 @@
+plugins {
+    id 'com.bmuschko.docker-remote-api'
+}
+
+evaluationDependsOn ':docker-java-and-python'
+evaluationDependsOn ':deephaven-jpy'
+evaluationDependsOn ':deephaven-wheel'
+evaluationDependsOn ':deephaven2-wheel'
```

Comment:
eesh i wish we had dependencies instead of this stringy goo you hope you never forget...

Reply:
I *think* the good news is that the gradle build will just fail to work if an evaluationDependsOn is missing. That said, it's not necessarily consistently failing - but hopefully it's obvious enough to us doing "things" in gradle how to fix it.